templates/jenkins_agent.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
This is a script used by systemd - I'm thinking whether converting this executable into Python would benefit much given that we're just executing java command.
We're using systemd so that we have it as a running service and charm manages the service rather than a single execution of a process.
Would this be a reasonable explanation?